src/tests/ftest/daos_test/suite.py

```diff
@@ -302,6 +302,22 @@ def test_daos_rebuild_simple(self):
         """
         self.run_subtest()
 
+    def test_daos_rebuild_simple_interactive(self):
+        """Jira ID: DAOS-1568
+
+        Test Description:
+            Run daos_test -v --rebuild_interactive
+
+        Use cases:
+            Core tests for daos_test
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium,provider
+        :avocado: tags=daos_test,daos_core_test
```

Comment:
We aren't always on top of this, but since this is a weekly test, we should consider: how/when will this run in a PR?
Tagging with `rebuild` is probably the simplest so people can use `Features: rebuild` without thinking about THIS test in particular.
Anyway, I'll let you decide
```suggestion
        :avocado: tags=daos_test,daos_core_test,rebuild
```

Reply:
Less urgency I agree but FWIW you can add them if you find it helpful